src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -73,25 +73,25 @@ class DriftTable extends Component {
     }
 
     renderRowData(fact, systems) {
-        let td = [];
+        let tr = [];
 
-        td.push(<td>{ fact.name }</td>);
-        td.push(<td className="fact-state"><StateIcon factState={ fact.state }/></td>);
+        tr.push(<th>{ fact.name }</th>);
+        tr.push(<th className="fact-state"><StateIcon factState={ fact.state }/></th>);
```

Comment:
Why change these `table data` cells to `table header` cells?

Reply:
It must have carried over from my horizontal scroll PR. I must have made a mistake somewhere when creating this branch. I'll revert it.